3 a 1

uzzelk opened the pull request.

MichaelGNairn reviewed (commented):
[Michael Nairn Code review for 3-a-1.docx](https://github.com/ONSdigital/sdg_data_updates/files/10839677/Michael.Nairn.Code.review.for.3-a-1.docx)

Review attached.

uzzelk commented:
Thanks for reviewing this so quickly @MichaelGNairn 😄 I've addressed your comments - added point 8 in the README.md and also added extra text to explain the production of multiple csvs (I originally tried joining them all sorts of ways but none of them were working as I wanted them too, so google suggested that option which seems to work pretty well!). 

I've left the country of birth for now, as I looked at a few other indicators and some report them as born in UK/not born in UK (e.g., 3.1.1, 3.3.2, 4.3.1), then others list the specific country of birth (e.g., 3.a.1, 3.2.2), but would be pretty easy to aggregate to report born in UK/not born in UK instead if you think this is a better way to report it?

As for discrepancies picked up in the QA file, they are for 2019 so I'm guessing there has been revisions to the back series, but I am going to update this indicator once the automation has been merged so can explore further then!
